dango/dex/src/core/order_filling.rs

```diff
@@ -108,28 +108,23 @@ fn fill_bids(
             fee_base,
             fee_quote,
             clearing_price,
-        });
-
-        if volume.is_zero() {
-            break;
-        }
```

Comment:
This check is actually unnecessary, because now we only collect orders that are matched (partially or fully) into the `bids` and `asks` vectors. There shouldn't be a case where there is an order in the `bids`/`asks` vector but there's not enough volume to fill it.